content/raw/chat/online_users.md

```diff
@@ -61,10 +68,13 @@ preconditions:
 
 **Status Selector**
 
-- The user MUST be able to select his state as 'online' or 'offline'
-- If the user has 'online' as selected, then they MUST be shown online in the communities member list.
-- If the user has 'offline' as selected, then they MUST be shown offline for everyone within 5 minutes regardless of activity, even if the user sends new messages.
+- The user MUST be able to select his state as 'always online' or 'inactive' or 'set status automatically'
+- If the user has 'always online' as selected, then they MUST be shown online in the communities member list.
+- If the user has 'inactive' as selected, then they MUST be shown offline for everyone within 5 minutes regardless of activity, even if the user sends new messages.
+- If the user has selected `Set status automatically`, then they MUST be shown online in the communities member list.
 
 ## Notes
 
 - under the hood: a user that has the online state sends a ping to a special community channel, if the user selects offline state, then they stop sending this ping, and it's assumed by other clients the user went offline if they don't receive another ping within 5 minutes.
```

Comment:
This should be:  

- under the hood: a user that has the online state (as a result of having 'Set status automatically' selected) sends a ping to a special community channel.   If the user stops interacting with Status then these pings stop being sent, and it's assumed by other clients the user went offline if they don't receive another ping within 5 minutes.